tests/e2e/medic-conf-tests.js

```diff
@@ -0,0 +1,45 @@
+const _ = require('lodash');
+const {expect} = require('chai');
+const util = require('util');
+const { API_PORT }=require('../constants');
+const exec = util.promisify(require('child_process').exec);
+
+const actions = ['compile-app-settings','backup-app-settings','upload-app-settings','convert-app-forms',
+  'convert-collect-forms','convert-contact-forms', 'backup-all-forms','upload-app-forms','upload-collect-forms',
+  'upload-contact-forms', 'upload-resources','upload-branding','upload-partners','upload-custom-translations',
+  'upload-privacy-policies', 'delete-all-forms'];
+
+describe('medic-conf supported actions', () => {
+  beforeAll( () =>{
+    // Change the directory
+    try {
+      process.chdir('config/default');
```

Comment:
Use ```process.chdir(`${__filenameconfig}../../config/default');``` to change your directory. Then there is no guess about where you are located.